react-basic-practice/src/App.tsx

```diff
@@ -1,61 +1,13 @@
 import React from "react";
-import { Box, ChakraProvider } from "@chakra-ui/react";
+import { ChakraProvider } from "@chakra-ui/react";
 import { theme } from "@themes";
-import { Table, TableColumn } from "@components";
-import { ICustomer } from "@types";
-
+import { Dashboard } from "@pages";
 
 const App: React.FC = () => {
-  {/* TO DO: This is a mock data, I'll update later*/}
-  const data: ICustomer[] = [
-    {
-      name: "Ann Culhane",
-      description:
-        "Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus nobis non obcaecati reiciendis pariatur. Perferendis pariatur labore, atque mollitia veniam voluptatum cum deserunt facilis at enim, fuga, provident inventore rem.",
-      status: "Open",
-      rate: "$70.00",
-      balance: "-$270.00",
-      deposit: "$500.00",
-      id: 1,
-    },
-    {
-      name: "user name",
-      description:
-        "Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus nobis non obcaecati reiciendis pariatur. Perferendis pariatur labore, atque mollitia veniam voluptatum cum deserunt facilis at enim, fuga, provident inventore rem.",
-      status: "Paid",
-      rate: "$70.00",
-      balance: "$270.00",
-      deposit: "$500.00",
-      id: 2,
-    },
-    {
-      name: "user name",
-      description:
-        "Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus nobis non obcaecati reiciendis pariatur. Perferendis pariatur labore, atque mollitia veniam voluptatum cum deserunt facilis at enim, fuga, provident inventore rem.",
-      status: "Due",
-      rate: "$70.00",
-      balance: "-$270.00",
-      deposit: "$500.00",
-      id: 3,
-    },
-    {
-      name: "user name",
-      description:
-        "Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus nobis non obcaecati reiciendis pariatur. Perferendis pariatur labore, atque mollitia veniam voluptatum cum deserunt facilis at enim, fuga, provident inventore rem.",
-      status: "Inactive",
-      rate: "$70.00",
-      balance: "$270.00",
-      deposit: "$500.00",
-      id: 4,
-    },
-  ];
-
   return (
     <ChakraProvider theme={theme}>
-      <Box>
-        {/* TO DO: I imported it here just to see its demo, I will update it later*/}
-        <Table columns={TableColumn} data={data} />
-      </Box>
+      {/* TO DO: I imported it here just to see its demo, I will update it later*/}
```

Comment:
Do we still have any updates here? Why is there still a TODO comment at this time?

Reply:
i done remove it